easypost/requestor.py

```diff
@@ -11,33 +13,41 @@
 
 
 class Requestor:
-    def __init__(self, local_api_key=None):
+    def __init__(self, local_api_key: str = None):
         self._api_key = local_api_key
 
     @classmethod
-    def _objects_to_ids(cls, param):
+    def _objects_to_ids(cls, param: dict) -> dict:
         if isinstance(param, EasyPostObject):
             return {"id": param.id}
         elif isinstance(param, dict):
-            return {k: cls._objects_to_ids(v) for k, v in param.items()}
-        elif isinstance(param, list):
-            return [cls._objects_to_ids(v) for v in param]
+            data = {}
```

Comment:
With these changes here, have we tried to force code to run through them by making a fresh API call (no VCR)?

Reply:
Getting some errors, but seem to be unrelated to this change.